src/WorkItems/AzureDevOpsClientWrapper.cs

```diff
@@ -13,21 +13,20 @@
 using Microsoft.VisualStudio.Services.WebApi.Patch.Json;
 using Newtonsoft.Json;
 
-namespace Microsoft.WorkItemFiling
+namespace Microsoft.WorkItems
 {
     /// <summary>
     /// Represents an Azure DevOps project in which work items can be filed.
     /// </summary>
-    public class AzureDevOpsClientWrapper : FilingClient
+    public class AzureDevOpsClientWrapper: FilingClient
```

Comment:
>AzureDevOpsClientWrapper [](start = 17, length = 24)

`AzureDevOpsFilingClient`. See my comment in GitHubClientWrapper.cs for more thoughts on this.